pkg/hive/reconciler_metrics.go

```diff
@@ -148,12 +148,12 @@ func (m *reconcilerMetricsImpl) IncrementalReconciliationDuration(moduleID cell.
 }
 
 // IncrementalReconciliationErrors implements reconciler.Metrics.
-func (m *reconcilerMetricsImpl) IncrementalReconciliationErrors(moduleID cell.FullModuleID, errs []error) {
+func (m *reconcilerMetricsImpl) IncrementalReconciliationErrors(moduleID cell.FullModuleID, newErrors, currentErrors int) {
 	if m.m.IncrementalReconciliationCurrentErrors.IsEnabled() {
-		m.m.IncrementalReconciliationCurrentErrors.WithLabelValues(LabelModuleId, moduleID.String()).Set(float64(len(errs)))
+		m.m.IncrementalReconciliationCurrentErrors.WithLabelValues(LabelModuleId, moduleID.String()).Set(float64(currentErrors))
 	}
 	if m.m.IncrementalReconciliationTotalErrors.IsEnabled() {
-		m.m.IncrementalReconciliationCurrentErrors.WithLabelValues(LabelModuleId, moduleID.String()).Add(float64(len(errs)))
+		m.m.IncrementalReconciliationCurrentErrors.WithLabelValues(LabelModuleId, moduleID.String()).Add(float64(newErrors))
```

Comment:
I think there is a bug here. We are checking if `IncrementalReconciliationTotalErrors` is enabled, but then adding to `IncrementalReconciliationCurrentErrors` which we set above. Seems like a copy-paste error. I think we should be adding to `IncrementalReconciliationTotalErrors` here. Can this be fixed in this PR as well, or should I submit a seperate fix?

Reply:
@dylandreimerink please submit in a follow up PR as I believe that `IncrementalReconciliationTotalErrors` should probably be `newErrors` + `currentErrors`?